clang/test/CodeGen/debug-info-debug-info-type.cpp

```diff
@@ -0,0 +1,14 @@
+// RUN: %clang -target x86_64-linux -g -S -emit-llvm -o - %s | FileCheck %s
+// RUN: %clang_cc1 -verify -DMISMATCH %s
```

Comment:
I think we should split this into two test: one for testing what debug info is generated (here in CodeGen) and another one for testing diagnostic behavior in Sema. The CodeGen test should be clean of diagnostics.

Another test I think we should add would be testing qualifier mismatches (where the bit-field is `const`-qualified to make sure that doesn't show up as a mismatch.

Reply:
Tests are split now. `const` case is now checked and working as expected (qualifiers are ignored).